data/projects.json

```diff
@@ -228,5 +228,15 @@
         "level": "intermediate",
         "site": "https://brave-mirzakhani-8081ab.netlify.app/",
         "tags": "vuejs, pico css, netlify"
+    },
+    {
+        "name": "codeet",
+        "display": "simple social page for coders",
+        "description": "A simple social page for coders.",
+        "author": "williamneves",
+        "category": "website",
+        "level": "intermediate",
+        "site": "https://codeet.code/",
```

Comment:
Please check the site URL, due to I could not load the page.
<img width="637" alt="image" src="https://user-images.githubusercontent.com/37890026/160124584-4177036f-ac10-4f3b-99ca-37b7f30048d2.png">